theme/templates/base.html

```diff
@@ -44,6 +45,7 @@
 <script src='{% static 'js/jquery-ui.js' %}'></script>
 <script src="{% static "js/bootstrap.3.0.2.min.js" %}"></script>
 <script src="{% static "js/bootstrap-extras.js" %}"></script>
+<script src="{% static "js/jQDateRangeSlider-min.js" %}"></script>
```

Comment:
Also will be used later for range slider, since I deleted all the related code for range slider part. I just removed this library for this release